sos/plugins/openstack_ironic.py

```diff
@@ -135,4 +148,24 @@ def setup(self):
             self.add_journal(units="openstack-ironic-discoverd")
             self.add_journal(units="openstack-ironic-discoverd-dnsmasq")
 
-# vim: set et ts=4 sw=4 :
+        # ironic-discoverd was renamed to ironic-inspector in Liberty
+        self.conf_list.append('/etc/ironic-inspector/*')
+        self.conf_list.append(self.var_puppet_gen + '/etc/ironic-inspector/*')
+        self.add_copy_spec('/etc/ironic-inspector/')
+        self.add_copy_spec(self.var_puppet_gen + '/etc/ironic-inspector/')
+        self.add_copy_spec('/var/lib/ironic-inspector/')
```

Comment:
Can't be here some big amount of data in some (unexpected) use case that sosreport will all copy and create ridiculously huge tarball?

Reply:
I'm not sure we even store anything there right now :) In the near future, there will be some amount of really small (several bytes) files, but that's it.